### DISCUSSION TITLE
Placeholder text usage suggestions

### DISCUSSION THREAD

**Discussion by sang-yeo** [twilio-labs/paste#1009] [Q&A] [answered]
Hello, 
What would you put a placeholder for the Value input in this case? 
<img width="459" alt="Screen Shot 2020-12-07 at 5 31 08 PM" src="https://user-images.githubusercontent.com/67388401/101429231-499b0980-38b7-11eb-871a-512727f15a95.png">
There’s no Field selected and Field contains Error code, multiple types of SIDs, From and To phone numbers, Product and Webhook. 

I was playing with something like this but the area is limited so I can't put all the examples. 
<img width="352" alt="Screen Shot 2020-12-07 at 6 11 21 PM" src="https://user-images.githubusercontent.com/67388401/101429398-b57d7200-38b7-11eb-98fd-26f147509059.png">

**Comment by sang-yeo** [comment]
Addition to this default placeholder I also created dynamic placeholder examples(see below). But I'm still not sure the default placeholder text delivers a clear message to users. 
<img width="2016" alt="Screen Shot 2020-12-08 at 10 17 20 AM" src="https://user-images.githubusercontent.com/67388401/101524515-df2dac00-393e-11eb-986d-4a49531ba5be.png">

**Comment by serifluous** [comment, marked as answer]
Hey @sang-yeo ! The placeholder text is a completely optional thing you can add to an input, so you don't actually even need it. For that reason, we don't have a strong recommendation for what content you should put there (aside from what's written in the [input composition guidelines](https://paste.twilio.design/components/input#optional-compositional-elements), which you've pretty much followed here), and it disappears when a user enters text anyway. If the placeholder content _is_ important enough to the customer, it's probably worth placing somewhere else on the screen.
